Firebase Push Notifications.md

```diff
@@ -0,0 +1,88 @@
+# Firebase push notifications
+
+Table of contents:
+
+1. [General](#general)
+2. [Setup](#setup)
+3. [Development approaches](#development-approaches)
+4. [Choosing right development approach](#choosing-development-approach)
+
+## General
+In this chapter we will explain few approaches for push notifications implementations and theirs PROS and CONS.
+
+There are many services which provides us with push notifications, but we are using Firebase because of its simplicity, reliability and because it's free.
```

Comment:
I wouldn't say that we are using it because it's free. Is there someting we could pay that would be better? If not, I would leave that out.